GOVERNANCE.md

```diff
@@ -1,30 +1,63 @@
+  the existing senior maintainers will confer and decide whether to grant maintainer status or not.
+  We make no guarantees on the length of time this will take, but 2-3 months is the approximate
   goal.
 
-## Your organization is currently a committer
+## Your organization is currently a maintainer
 
-* First decide whether your organization really needs more people with commit access. Valid reasons
-  are "blast radius", a large organization that is working on multiple unrelated projects, etc.
-* Contact a senior committer for your organization and express interest.
-* Start doing PRs and code reviews under the guidance of your senior committer.
-* After a period of 1-2 months the existing senior committers will discuss granting "standard"
-  commit access.
-* "Standard" commit access can be upgraded to "senior" commit access after another 1-2 months of
-  work and another conference of the existing senior committers.
+* First decide whether your organization really needs more people with maintainer access. Valid
+  reasons are "blast radius", a large organization that is working on multiple unrelated projects,
+  etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard"
+  maintainer access.
+* "Standard" maintainer access can be upgraded to "senior" maintainer access aft
+
+## Maintainer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+* In general continue to be willing to spend at least 25% of ones time working on Envoy (~1.25
+  business days per week).
+* Note that currently the above is performed by all maintainers in a best-effort/haphazard way. In
+  the future we may decide to move to an "on-call" rotation.
+
+## When does a maintainer lose maintainer status
+
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they
+should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of
+the maintainers per the voting process below.
+
+# Conflict resolution and voting
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out
+between the persons involved. If a dispute cannot be decided independently, the maintainers can be
+called in to decide an issue. If the maintainers themselves cannot decide an issue, the issue will
+be resolved by voting. The voting process is a simple majority in which each senior maintainer
+receives two votes and each normal maintainer receives one vote.
+
+# Adding new projects to the envoyproxy GitHub organization
+
+New projects will be added to the envoyproxy organization via GitHub issue discussion in one of the
```

Comment:
I can't think of a concrete example, but what would happen if someone wants to add a project that does not logically fit as an issue in any of the existing projects. Would it be worth it to have a repo for RFCs of this type?

Reply:
Yeah I think that's a good idea, though I think they could email envoy-dev@, etc. so maybe let's see if this becomes a real sisue?